GitHubBranches.ps1

```diff
@@ -192,11 +536,28 @@ filter Add-GitHubBranchAdditionalProperties
 
         if (-not (Get-GitHubConfiguration -Name DisablePipelineSupport))
         {
-            $elements = Split-GitHubUri -Uri $item.commit.url
+            if ($null -ne $item.url)
+            {
+                $elements = Split-GitHubUri -Uri $item.url
+            }
+            else
+            {
+                $elements = Split-GitHubUri -Uri $item.commit.url
+            }
```

Comment:
You're technically accessing `$elements` out-of-scope below in the `Add-Member`.  We can simplify this statement and ensure it's defined at the same scope as `Add-Member`.

```suggestion
            $elements = Split-GitHubUri -Uri $item.url
            if ($null -eq $item.url)
            {
                $elements = Split-GitHubUri -Uri $item.commit.url
            }
```

Reply:
Ugly code, but life is too short. Done.

Reply:
Ok, let's revert this one to what you originally had.  I hadn't read the code closely enough the first time (missed that you were assigning from a function result as opposed to the URL itself.  The first call will fail if `$item.url` is $null (since `Split-GitHubUri` has `[ValidateNotNullOrEmpty()]` on the `$Uri` parameter.

Reply:
Done